### ISSUE TITLE
We need the ability to export identities from connection specific stores programmatically

### ISSUE THREAD

**Issue by davidkel** [hyperledger/composer#1669]
This is only required for playground as part of the ID Card work. At this point in time we don't need to provide a cli capability to the end user. 
The proposal is an API on AdminConnection as follows
1. Add exportIdentity to the connectionmanager.js `ConnectionManager.exportIdentity(connectionProfile, connectionOptions, id)`
2. Expose this in adminconnection in a similar manner to importIdentity
Add the implementation required for HLFV1 to HLFConnectionManager.js in the composer-connector-hlfv1 code

also note the importIdentity doesn't take a public key it takes a public certificate so It would be good to rectify the code and doc to reflect this.

Prototype example code in the HLFConnectionManager.js that can extract the crypto-material is here
```
    exportIdentity(connectionProfile, connectionOptions, id) {
        const client = HLFConnectionManager.createClient();
        return this._setupWallet(client, connectionOptions.wallet, connectionOptions.keyValStore)
            .then(() => {
                return client.getUserContext(id, true);
            })
            .then((user) => {
                return {
                    publicCertPEM: user.getIdentity()._certificate,
                    privateKeyPEM: user.getSigningIdentity()._signer._key.toBytes()
                };
            });
    }
```
I've spoken to Jim about the api and he is happy with that approach and I will raise a Jira for the missing getters so this approach isn't going to require changing.

**Comment by sstone1** [comment]
Agree on the rename from "public key", but it should just be "certificate" not "public certificate".

**Comment by davidkel** [comment]
No special reason for PEM being there apart from being more descriptive. We can drop the PEM text in the property names

**Comment by bestbeforetoday** [comment]
Resolved by #1427.
